scripts/build.py

```diff
@@ -163,3 +163,8 @@ def get_cache_slug():
     CSS_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', 'css', '')
     CSS_OUT_DIR = os.path.join(BUILD_DIR, 'css', '')
     copy_files_source_to_target(CSS_SRC_DIR, CSS_OUT_DIR)
+
+    # Copy core/templates/prod/head/ to build/[cache_slug]/templates/head/
+    TEMPLATES_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
+    TEMPLATES_OUT_DIR = os.path.join(BUILD_DIR, 'templates', 'head', '')
```

Comment:
I meant we could (will have to test once) move template files as well to build and then read them from there instead of /prod/head.